### PR TITLE
LPS-164614

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-164614

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss [hunk 1]
```diff
@@ -498,29 +565,32 @@
 		min-height: 500px;
 	}
 
-	.report-cards-area {
+	&--cards-area {
+		margin-left: 20px;
 		transition: all 0.3s cubic-bezier(0.25, 0.11, 0.25, 1);
-		width: 806px;
+		width: 80%;
+		width: 808px;
```
Comment (rebsilva) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss:572]: I think here you only meant to use the `width: 808px;` and not the 80% one, could you verify please?

**Review by rebsilva — CHANGES_REQUESTED** [review]
Hello @alinedoleron, really great job on this task! I've just found two minor issues that need to be changed, let me know if you have any questions! :cherry_blossom:

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss [hunk 2]
```diff
@@ -53,55 +53,128 @@
...
+		transition: box-shadow 0.15s ease-in-out;
+	}
 
-				.card-item {
-					width: 808px;
-				}
-			}
+	.menubar-vertical-expand-lg.menubar-transparent .menubar-collapse {
+		background-color: #fff;
+		border-color: #e7e7ed;
```
Comment (rebsilva) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss:109]: Wouldn't be better to use the colors variables that we already have on code like $white and $secondary-l3? To keep the pattern we've been doing and make the code more readable.

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss [hunk 3]
```diff
@@ -53,55 +53,128 @@
...
+		border-radius: 0.25rem;
+		box-shadow: 0 1px 5px -1px rgb(0 0 0 / 30%);
+	}
 
-			.report-cards-shortcut {
-				display: block;
-			}
+	.menubar-vertical-expand-lg.menubar-transparent {
+		background-color: #fff;
```
Comment (rebsilva) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss:115]: Same as above, change this to $white

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss [hunk 4]
```diff
@@ -53,55 +53,128 @@
...
+		.menubar-toggler {
+			display: block;
 		}
+	}
 
-		@media (min-width: 1280px) {
-			.report-cards-area {
-				width: auto;
+	.menubar-vertical-expand-lg.menubar-transparent .menubar-toggler {
+		color: #272833;
```
Comment (rebsilva) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss:100]: If we have a variable for this color in 'atlas-variables', please change this to keep the pattern we've been doing

**Review by rebsilva — APPROVED** [review]
Hi @alinedoleron, thanks for your work on this! I've asked some changes, but since it's only some enhancement you can send them in another PR so this one can move forward as soon as possible! So it's good to go :tada:

**Comment by rebsilva** [comment]
ci:forward

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**